lib/operator.d.ts

```diff
@@ -1,9 +1,9 @@
+export function arbitrary(version: string, other: string): boolean;
```

Comment:
```suggestion
```
not exported

https://github.com/renovatebot/pep440/blob/5fb26d3568620c980f92a09d7e32561cae314613/lib/operator.js#L3-L19

Reply:
We can ignore the string exports, as they can't really imported.

Reply:
There are tests which require the string operators.
So leaving them as exported for now.

Reply:
We need those string operator exports for tests.